packages/edit-site/src/components/add-new-template/new-template-part.js

```diff
@@ -36,8 +36,7 @@ export default function NewTemplatePart( { postType } ) {
 		} );
 
 		// Navigate to the created template part editor.
-		window.location.search = addQueryArgs( '', {
-			page: 'gutenberg-edit-site',
+		window.location.href = addQueryArgs( window.location.href, {
```

Comment:
Would it be easier if we use `window.location.pathname`?

```suggestion
		window.location.href = addQueryArgs( window.location.pathname, {
```

Reply:
Most links in WP Admin use absolute, not relative, URLs, so I say let's do that. (Also I don't want to wait for CI again 😂.)